template_feed/Microsoft.DotNet.Common.ItemTemplates/content/BuildTargets/.template.config/ide.host.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json.schemastore.org/ide.host",
+  "itemHierarchyPaths": ["General"],
+  "appliesTo": ".NET",
```

Comment:
This shouldn't be necessary for Visual Studio. I have an item template that has no appliesTo and it is visible